app/components/case-studies-components/CaseStudiesTitleText.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+interface CaseStudiesTitleTextProps {
+  wordOne: string;
+  wordTwo: string;
+  wordThree: string;
+}
+
+const CaseStudiesTitleText: React.FC<CaseStudiesTitleTextProps> = ({
+  wordOne,
+  wordTwo,
+  wordThree,
+}) => {
+  return (
+    <div className="flex flex-col lg:flex-row">
```

Comment:
Don't forget the importance of HTML semantics. Consider changing this `<section>` to a `<header> `element.

Reply:
thanks. i have changed to a section

Reply:
thanks. i have changed to a section